src/main/fc/fc_init.c

```diff
@@ -459,6 +447,18 @@ void init(void)
 
     rxInit();
 
+#ifdef OSD
```

Comment:
Can you put in a comment saying the OSD should be initialised after the gyro. Otherwise someone might change it back again in future.